Sources/BluetoothServices/Characteristics/TemperatureMeasurement.swift

```diff
@@ -0,0 +1,130 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import NIO
+import SpeziBluetooth
+
+
+/// A temperature measurement.
+public struct TemperatureMeasurement {
```

Comment:
If this is from some sort of standard it would be gerat if we could link to it in the documentation, e.g.:
```
/// This struct is part of the Swift types that implement the Bluetooth [Health Thermometer Service 1.0](https://www.bluetooth.com/specifications/specs/health-thermometer-service-1-0).
```